### DISCUSSION TITLE
Allelic frequency & mutation site coverage

### DISCUSSION THREAD

**Discussion by bajwamoneeb** [jodyphelan/TBProfiler#397] [General]
Hi Jody,

I was wondering if you had any detailed data on studies you may have done on ideal allelic frequency & mutation site coverage for TB-Profiler (defaults are 90% and 10x for your pipeline I see). @jodyphelan

**Comment by jodyphelan** [comment]
Hi @bajwamoneeb 

By default tb-profiler will report all mutations that the variant caller finds and will use variants > 10% for resistance prediction. With depth it is 10x as you noted. 

These are pretty arbitrary to be honest and might need to be adjusted according to an individual study or protocol. We have done some internal validation of these cutoffs on a small set of samples, however no large scale study was performed.
